ts/modules/dotplot.src.ts

```diff
@@ -129,7 +129,7 @@ seriesType<Highcharts.DotplotSeries>('dotplot', 'column', {
                 }
 
                 itemY = point.y;
-                yTop = pick(point.stackY, point.y);
+                yTop = pick(point.stackY, point.y) as any;
```

Comment:
This should not be done in that way, because not the pick is the problem here but the arguments. If `any` has to be used at all, then something similar to `pick<number>(a, b as any)` is better, because it points to the problem source.